keyboards/handwired/daskeyboard/daskeyboard4/readme.md

```diff
@@ -0,0 +1,72 @@
+# daskeyboard4
+
+![daskeyboard4pro](https://i.imgur.com/Y3xT9Zu.jpeg)
+
+![PCB](https://i.imgur.com/UfzVHcj.jpeg)
+
+*Reverse engineered PCB definition of daskeyboard 4 (Professional)*
+
+* Keyboard Maintainer: [Commander1024](https://github.com/Commander1024)
+* Hardware Supported: *The PCB, rotary encoder, LEDs*
+* Hardware Availability: *https://www.daskeyboard.com/de/daskeyboard-4-professional/*
+
+I used a STM32 development board to replace the keyboard's destroyed controller. This keyboard definition is here to enable to painlessly use the OEM PCB in a custom keyboard, as the rows and columns are placed in an unintuitive way.
```

Comment:
Could you clarify this line, please? Which development board? Perhaps a sourcing link?

Reply:
Does [fe14331](https://github.com/qmk/qmk_firmware/pull/18675/commits/fe143310b87e1cb894cb6b6a61588c247abd53fd) clarify things?